tests/end2end_test/__init__.py

```diff
@@ -0,0 +1,135 @@
+import time
+import logging
+from tests.end2end_test.end_to_end_test_helper import (
+    simple_query
+)
+from data_pipeline.data_store.s3_data_service import (
+    delete_s3_object
+)
+LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=too-few-public-methods, missing-class-docstring
+# pylint: disable=too-many-instance-attributes
+class DataPipelineCloudResource:
```

Comment:
I wonder whether the class name could be improved. No immediate suggestion but had to go through a bit to understand it a bit more. It seems to be specific for the end to end tests. (And tables are getting truncated which the caller should maybe be aware of)